client/js/_author/components/common/tiny_wrapper.jsx

```diff
@@ -89,7 +90,11 @@ export default class TinyWrapper extends React.Component {
           id={`${this.id}-tinymce`}
           content={this.props.text}
           config={this.tinyMCEConfig()}
-          onBlur={(e) => { this.props.onBlur(e.target.getContent(), e.target.isDirty()); }}
+          onBlur={(e) => {
+            const content = e.target.getContent();
+            const isChanged = e.target.isDirty() || _.isEmpty(content);
```

Comment:
That doesn't seem to be true. When I add content and then make the question text empty, it is correctly updating. Where are you seeing that bug?